.buildkite/dagster-buildkite/dagster_buildkite/steps/trigger.py

```diff
@@ -28,6 +29,12 @@ def trigger_step(
         "trigger": pipeline,
         "label": f":link: {pipeline} from dagster@{commit}",
```

Comment:
Should we also source `commit` from `DAGSTER_COMIT_HASH` instead of the subprocess call?